yapapi/runner/__init__.py

```diff
@@ -179,6 +196,13 @@ async def map(
         worker: Callable[[WorkContext, AsyncIterator["Task"]], AsyncIterator[Tuple["Task", Work]]],
         data,
     ):
+        """Run computations on providers.
+
+        :param worker: a function that takes a WorkContext object and a list o tasks,
+                       adds commands to the context object and yields committed comments
+        :param data: a list of Task objects to be computed on providers
```

Comment:
A "list" is perhaps too specific, maybe a "sequence" or "iterable", if we wanted to be super correct.

Reply:
Fixed.